catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -376,13 +413,25 @@ protected void printStatus(long count) {
             return null;
         }
 
-        List<MetacardTransformer> metacardTransformerList = new ArrayList<MetacardTransformer>();
+        List<MetacardTransformer> metacardTransformerList = new ArrayList<>();
         for (int i = 0; i < refs.length; i++) {
-
             metacardTransformerList.add((MetacardTransformer) bundleContext.getService(refs[i]));
         }
 
         return metacardTransformerList;
     }
 
+    private QueryResponseTransformer getZipCompression() {
+        List<QueryResponseTransformer> queryResponseTransformerList = getAllServices(
+                QueryResponseTransformer.class);
+        QueryResponseTransformer zipTransformer = null;
+        for (QueryResponseTransformer queryResponseTransformer : queryResponseTransformerList) {
+            if (queryResponseTransformer.getClass()
+                    .getCanonicalName()
+                    .contains(ZIP_COMPRESSION)) {
```

Comment:
A refactoring that changes the name of the class would break this code.  I would suggest refactoring the ZipCompression/ZipDecompression classes into services that can then be injected both here and in the zip transformers.